draft-ietf-taps-interface.md

```diff
@@ -1364,8 +1364,9 @@ Representation of Security Parameters in implementations should parallel that ch
 
 Security decisions, especially pertaining to trust, are not static. Once configured,
 parameters may also be supplied during connection establishment. These are best
-handled as client-provided callbacks. Security handshake callbacks that may be
-invoked during connection establishment include:
+handled as client-provided callbacks. 
+Callbacks block the progress of a connection, which distinguishes them from other Events in the transport system. How callbacks and events are implemented is specific to each implementation.
```

Comment:
```suggestion
Callbacks block the progress of a Connection's establishment, which distinguishes them from other Events in the transport system. How callbacks and events are implemented is specific to each implementation.
```

Reply:
I would prefer to leave the connection on lower case (as I am not sure that the connection object we refer to in upper case has/makes progress, but something in the underlaying transport system).

 ```suggestion
Callbacks block the progress of the connection establishment, which distinguishes them from other Events in the transport system. How callbacks and events are implemented is specific to each implementation.
```

Reply:
ok, this makes sense to me